cmd/open.js

```diff
@@ -40,7 +31,11 @@ exports.handler = (argv) => {
 };
 
 function getService(argv, res) {
-    if (argv.npms) { return `https://npms.io/search?term=${argv.package}`; }
-    if (argv.npm) { return res.body.collected.metadata.links.npm; }
-    return res.body.collected.metadata.links.repository || res.body.collected.metadata.links.npm;
+    const links = res.body.collected.metadata.links;
+
+    if (argv.link) {
```

Comment:
I know but specifying `default` in the yargs option the user knows what is the default value in the `npms open -h`. Additionally, it's still backwards compatible if you do what I suggested, because argv.target will always have a value.